EC-84: Restructure eZ Commerce doc, part 2 (#2)
ezsystems/commerce-documentation: docs/getting_started/installation_and_configuration_faqs.md

# Installation and configuration FAQ

> ## What configuration is required in order to communicate with ERP?

Reviewer: "in order" is redundant throughout

> ## What configuration is required in order to communicate with ERP?

Reviewer 2: ```suggestion
## What configuration is required to communicate with ERP?
```

> In order to communicate with ERP you have to make sure, that:

Reviewer: To communicate with the ERP system, make sure that:

> - You have configured the correct webconnector URL: `webconnector_url: ''`

Reviewer: For future consideration - If we followed STE recommendations, we would not use present perfect and resort to simple past only. This increases chances for content reuse and legibility if the reader is not English-speaking.
webconnector is name of a proprietary piece of software. Is it what you mean here? Or should it be "web connector"?

ln -s ../../src/<Company>/Bundle/ProjectBundle/Resources/xsl
```

- Make sure that you have the correct version of `vendor/moyarada`.

## Images in shop are not converted. What should I check?

Make sure that you have set up correct rights for the image folder:

``` bash
sudo chmod -R g+w web/var/ecommerce/storage/
```

## I am getting an error after installation

`[Defuse\Crypto\Exception\BadFormatException] Encoding::hexToBin() input is not a hex string`

> You need to generate a secret using `./vendor/defuse/php-encryption/bin/generate-defuse-key`  

Reviewer: I would consider writing "You must generate a secret by using the following utility:" or even "Generate a secret by using the following utility:"

> You need to generate a secret using `./vendor/defuse/php-encryption/bin/generate-defuse-key`  

Reviewer 2: ```suggestion
Generate a secret using `./vendor/defuse/php-encryption/bin/generate-defuse-key` and
```

> You can configure it in `parameters.yml`:

Reviewer: I would consider writing "Then configure the secret in the `parameters.yml` file:"

("You can" implies that there are other places where the secret can be configured.)

> You can configure it in `parameters.yml`:

Reviewer 2: ```suggestion
configure it in `parameters.yml`:
```
